src/app/components/dataset-details/dataset-details.component.ts

```diff
@@ -70,4 +72,12 @@ export class ItemDetailsComponent {
   isNotEmtpy() {
     return !!this.datasetDetails && Object.keys(this.datasetDetails).length > 0
   }
+
+  getDistributions(): any[] {
+    let distributions = this.datasetDetails["resources"];
+    if (!!distributions && Array.isArray(distributions.value)) {
+      return distributions.value;
+    }
+    return [];
```

Comment:
If `Array.isArray` is not needed, the whole function can be rewritten in a cleaner way:

`return this.datasetDetails?.resources?.value || [];`